src/main/resources/templates/pebt/householdList.html

```diff
@@ -16,7 +16,6 @@
           <div class="boxed-content text--centered">
             <strong th:text="#{household-list.your-household}"></strong>
             <ul class="subflow-list list--bulleted">
-              <li class="spacing-below-15" th:text="${inputData.firstName} + ' ' + ${inputData.lastName} + ' (' + #{household-list.thats-you} + ')'"></li>
```

Comment:
I think the logic should be something like:

Include the applicant's name if they answer "Yes" that they are in the household AND they are not applying for themselves (since in that case they will be included in the list of students).